src/utils/exportSQL/sentToGithub.js

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios';
+
+const TOKEN = 'YOUR_GITHUB_TOKEN';
```

Comment:
This is unacceptable. Tokens do not go in source files. Also this won't work in production